.circleci/config.yml

```
@@ -934,6 +934,7 @@ workflows:
                 - /^webflow-fundings-sync-.*/
                 - /^webflow-licenses-sync-.*/
                 - /^webflow-industries-sync-.*/
+                - /^content-repo-.*/
```

[boulder] Right now this is matching looking for a string to contain this substring`content-repo-`. This wouldn't catch the branch name.

This should be looking for an exact match.

Comment applies to all instances.
```suggestion
                - /^content-repo$/
```